### DISCUSSION TITLE
运行gocqhttp.bat文件显示乱码

### DISCUSSION THREAD

**Discussion by zcy8557** [Mrs4s/go-cqhttp#2136] [Q&A] [answered]
![屏幕截图 2023-05-10 092723](https://github.com/Mrs4s/go-cqhttp/assets/128271922/4180e5d6-8226-4b07-a6f6-9a7c9b396a3f)
运行后显示乱码，复制出来是正常的内容，怎么解决

**Comment by bilili-syy** [comment, marked as answer]
https://blog.csdn.net/qq_35038153/article/details/78430359
